src/components/Gallery.scss

```
@@ -0,0 +1,67 @@
+@use "../styles/_variables.scss" as *;
+
+.Gallery {
+  display: grid;
+  grid-gap: 0.625em;
+  grid-template-columns: repeat(auto-fit, minmax(12.5em, 1fr));
+  grid-auto-rows: 12.5em;
+  grid-auto-flow: dense;
+  padding: 0.625em;
+  height: 100vh;
+  overflow: scroll;
+
+  &__ImageContainer {
```

It looks like you're using [BEM naming](https://getbem.com/naming/), which I _love_, but it should still use kebab case, so:
```suggestion
  &__image-container {
```